### ISSUE TITLE
Missing download links for 'Depth_Metric3d' and  absolute depth 'Reconstruct' data

### ISSUE THREAD

**Issue by hitttttszz** [cvlab-stonybrook/multi-view-gte#1]
Thank you for your impressive work. I am currently trying to reproduce your training results using the official code, and I have successfully downloaded the Data and Annotations from the project webpage. 

However, when running train_multiview.py, I encountered a FileNotFoundError because the depth features are missing.
According to the code in data/dataset_multiview.py, the dataloader expects the following two directories which are currently not available in the main dataset download:
1.Depth_Metric3d/: (e.g., Depth_Metric3d/Shop/06_26/Cam1/0000.npy).
2.Reconstruct/scaleshift_metric3d/
I noticed the README.md in MVGT Dataset mentions:
"We will release data for other modalities (depth maps, scale and shift for absolute depth estimation) separately."

However, I could not find the download links for these separate modalities on the project webpage.Could you kindly share the download links for depth maps, scale and shift for absolute depth estimation separately.

Thank you very much for your time and help!
Best regards.

**Comment by qiaomu-miao** [comment]
Thanks for your interest.

You’re correct that the depth maps and scale/shift files are not included in the current public release. These were planned for a separate release, but I haven’t yet had time to properly organize and curate the files for public release. I will update the repository once this is done.

In the meantime, you may also generate depth maps using Metric3D and estimate absolute depth and scale/shift parameters with DUSt3R, following the procedures described in the supplementary material of the paper.

**Comment by keqizero** [comment]
> Thanks for your interest.
> 
> You’re correct that the depth maps and scale/shift files are not included in the current public release. These were planned for a separate release, but I haven’t yet had time to properly organize and curate the files for public release. I will update the repository once this is done.
> 
> In the meantime, you may also generate depth maps using Metric3D and estimate absolute depth and scale/shift parameters with DUSt3R, following the procedures described in the supplementary material of the paper.

Hi, thank you so much for the explanation. Do you have an estimated time for releasing the data? 

Meanwhile, following the description in the paper, I have a few questions: 

1. How did you incorporate the real-world calibration into the DUSt3R pipeline? 

2. What method was used to generate the mask for dynamic areas? 

3. Which Metric3D model was the best according to your experiments? Did you use the raw Metric3D output, or was there an additional refinement step?

4. What was your RANSAC residual threshold (epsilon) for the scale/shift estimation?

Thank you so much!

**Comment by qiaomu-miao** [comment]
> > Thanks for your interest.
> > You’re correct that the depth maps and scale/shift files are not included in the current public release. These were planned for a separate release, but I haven’t yet had time to properly organize and curate the files for public release. I will update the repository once this is done.
> > In the meantime, you may also generate depth maps using Metric3D and estimate absolute depth and scale/shift parameters with DUSt3R, following the procedures described in the supplementary material of the paper.
> 
> Hi, thank you so much for the explanation. Do you have an estimated time for releasing the data?
> 
> Meanwhile, following the description in the paper, I have a few questions:
> 
> 1. How did you incorporate the real-world calibration into the DUSt3R pipeline?
> 2. What method was used to generate the mask for dynamic areas?
> 3. Which Metric3D model was the best according to your experiments? Did you use the raw Metric3D output, or was there an additional refinement step?
> 4. What was your RANSAC residual threshold (epsilon) for the scale/shift estimation?
> 
> Thank you so much!

Hello,

I will release the depth estimations and scale/shift files by the end of next week. 

Regarding your questions:
1. Dust3r has this [demo code](https://github.com/naver/dust3r/blob/main/dust3r/demo.py) for getting the 3D reconstruction and the optimized depth estimations. For adding calibration parameters, you can refer to these two issues: https://github.com/naver/dust3r/issues/54#:~:text=Contributor-,I%20added%20it%20in%204a414b6,-Example%20usage%3A
https://github.com/naver/dust3r/issues/30#:~:text=If%20you%20know%20the%20camera%20poses%20in%20advance%2C%20you%20can%20set%20them%20with%20the%20following%3A

2. We just detected the persons in the images and manually overlaid the calibration cubes for the calibration images, and treated all the rest of the area as static.

3. From what I can recall, we used DINO2reg-ViT-giant2 for the Metric3D model, though I think smaller model like ViT-Large should not result in a large change in results.

4. We just used the default setting of RANSACRegressor in sklearn.

I will also try to release the script for reconstruction in Dust3r and the scale shift estimation soon.

Thank you.

**Comment by qiaomu-miao** [comment]
Hello,

I have uploaded the depth maps and scale shifts. Please check the Readme for the links.

**Comment by keqizero** [comment]
> Hello,
> 
> I have uploaded the depth maps and scale shifts. Please check the Readme for the links.

Thank you so much! The google drive link to the estimated scale shifts for Cross-view GTE is not shared publicly, could you check it please?
